firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+FindInPage.swift

```diff
@@ -5,11 +5,32 @@
 import Shared
 
 extension BrowserViewController {
-    func updateFindInPageVisibility(visible: Bool, tab: Tab? = nil) {
-        if visible {
-            if findInPageBar == nil {
-                setupFindInPage()
-            }
+    func updateFindInPageVisibility(isVisible: Bool, tab: Tab? = nil) {
+        // TODO: The find interactions for iOS 16 close themselves, so once the min deployment target is iOS 16,
```

Comment:
note: If we add a TODO, best practice is to also create a ticket and add the number, so we can back track

Reply:
Thank you! Ticket made #20807.